src/index.css

```diff
@@ -112,9 +112,104 @@ hr {
   color: var(--text);
 }
 
+.btn {
+  margin: 0.11rem;
+  box-shadow: rgba(0, 0, 0, 0.4) 0 2px 4px, rgba(0, 0, 0, 0.3) 0 7px 13px -3px, rgba(0, 0, 0, 0.2) 0 -3px 0 inset;
+}
+
+/* Progress circle */
+
+.book-progress {
+  display: flex;
+  margin: 0 30px;
+}
+
+.circle-wrap {
+  width: 60px;
+  height: 60px;
+  background: #fff;
+  border-radius: 50%;
+  border: 2px solid #e8e8e8;
+  margin-right: 10px;
+}
+
+.mask .fill {
+  clip: rect(0, 30px, 60px, 0);
+  background-color: var(--primary);
+}
+
+.mask.full,
+.circle .fill {
+  animation: fill ease-in-out 3s;
+  transform: rotate(180deg);
+}
+
+.circle-wrap .circle .mask,
+.circle-wrap .circle .fill {
+  width: 60px;
+  height: 60px;
+  position: absolute;
+  border-radius: 50%;
+}
+
+.circle-wrap .circle .mask {
+  clip: rect(0, 60px, 60px, 30px);
+}
+
+@keyframes fill {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(180deg);
+  }
+}
+
+.circle-wrap .inside-circle {
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  margin-top: 5px;
+  margin-left: 5px;
+  background: #fff;
+  z-index: 1;
+}
+
+.percentage {
+  font-size: 2rem;
+  margin-bottom: 0;
+  margin-top: 10px;
+}
+
+.completed {
+  margin-top: 0;
+  font-size: 0.875rem;
+  opacity: 0.5;
+}
+
+.current-chap {
+  font-size: 0.813rem;
+  font-weight: 300;
+  opacity: 0.5;
+  margin: 0;
+}
+
+.chap-num {
+  margin-top: 0.438rem;
+  font-size: 1rem;
+  font-weight: 300;
+  letter-spacing: -0.4px;
+}
+
+.book-update {
+  padding-left: 0.688rem;
+  padding-top: 0.73em;
+}
+
```

Comment:
- [ ] Good job so far your design is impressive  :clap: but please make sure that your navigation bar has the same background color `white` defined on the original template. And make sure that the spacing between the `chapter` and `updates progress` is the same as defined on the original template  It is very important to stick to the design as much as possible as a developer it is important to pay attention to details. :+1: 

#### Your design
![1](https://user-images.githubusercontent.com/6615394/209237649-391b367b-88fb-43fe-80c4-e8d1d664fcdd.jpg)

#### Original template
![bookstore2](https://user-images.githubusercontent.com/6615394/209238069-de07eeaf-94ee-41b2-a7e7-b0e0eb32ff5e.png)